blockchain/validate.go

```diff
@@ -803,6 +823,40 @@ func (b *BlockChain) checkBlockContext(block *btcutil.Block, prevNode *blockNode
 				return err
 			}
 		}
+
+		// If the CSV soft-fork isn't fully active, then we can skip
+		// the sequence locks checks below.
+		if csvState != ThresholdActive {
+			return nil
+		}
+
+		// Load all of the utxos referenced by the inputs for all
+		// transactions in the block don't already exist in the utxo
+		// view from the database. This view is required in order to
+		// properly evalulate the sequence locks (relative lock-time)
+		// for each input referenced within the block.
+		utxoView := NewUtxoViewpoint()
```

Comment:
Since we don't have a utxo cache this will end up double loading the utxos.  Also, this assumes the block is extended the main chain which may or may not be true.  We'll need to figure out a better place to do these checks and/or get the view earlier and pass it in to the function as well as down the call chain.